drv/i2c-devices/src/tps546b24a.rs

```diff
@@ -64,8 +76,9 @@ impl Tps546B24A {
 
 impl Validate<Error> for Tps546B24A {
     fn validate(device: &I2cDevice) -> Result<bool, Error> {
-        let expected = [0x54, 0x49, 0x54, 0x6B, 0x24, 0x41];
-        pmbus_validate!(device, IC_DEVICE_ID, expected)
+        let expected = &[0x54, 0x49, 0x54, 0x6B, 0x24, 0x41];
```

Comment:
This is _technically_ ASCII, but dunno if
```suggestion
        let expected = b"TITk$A";
```
is any clearer 😆

Reply:
Lol yeah I checked that one too and opted to leave it :joy: